tests/source.rs

```diff
@@ -29,7 +29,6 @@ extern crate regex;
 extern crate rls_analysis as analysis;
 extern crate shlex;
 extern crate tempdir;
-extern crate jsonapi;
```

Comment:
this change is technically unrelated to this PR, but was causing a warning so I tossed it in here.

Reply:
Might as well!